qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    guest_cmd = params['guest_cmd']
+    for x in range(2):
+        vm = env.get_vm(params["main_vm"])
+        if x >= 1:
+            params['serials'] = ''
```

Comment:
This will effect the default serials.

Reply:
Hi from polarion testcase:
. Boot guest A with
-chardev tty,id=serial0,path=/dev/ttyS0 -device isa-serial,chardev=serial0

 Boot guest B with
-serial /dev/ttyS0

It seem like guest B doesnt want any default serial

Reply:
@michiboo polarion test steps only list the command that related with the test scenario.
Framework have default serials which should not be effected.